test/models/organization_test.rb

```diff
@@ -126,4 +126,17 @@ def organization
   test 'related developments custom method responds correctly' do
     assert_respond_to(org, :developments)
   end
+
+  private
+
+  def webster
+    """
+      Lake Char­gogg­a­gogg­man­chaugg­a­gogg­chau­bun­a­gung­a­maugg
+      / Webster Lake, Webster, Massachusetts, United States of America
+    """.strip.gsub(/\s+/, ' ')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.